otherlibs/dune-rpc/private/exported_types.ml

```diff
@@ -179,10 +179,17 @@ module Diagnostic = struct
   ;;
 
   module Id = struct
-    type t = int
+    module T = struct
```

Comment:
Should be reverted

Reply:
Why? This is how we get the maps.

Reply:
You can move the map functor application to the caller. We want to reduce the dependence on stdune in that library.